conf/base.config

```diff
@@ -20,22 +20,25 @@ process {
 
     // Process-specific resource requirements
     withLabel:process_low {
-        cpus   = { check_max( 2     * task.attempt, 'cpus'   ) }
-        memory = { check_max( 12.GB * task.attempt, 'memory' ) }
-        time   = { check_max( 4.h   * task.attempt, 'time'   ) }
+        cpus   = { check_max( 2     * task.attempt, 'cpus'    ) }
+        memory = { check_max( 12.GB * task.attempt, 'memory'  ) }
+        time   = { check_max( 4.h   * task.attempt, 'time'    ) }
     }
     withLabel:process_medium {
-        cpus   = { check_max( 6     * task.attempt, 'cpus'   ) }
-        memory = { check_max( 36.GB * task.attempt, 'memory' ) }
-        time   = { check_max( 8.h   * task.attempt, 'time'   ) }
+        cpus   = { check_max( 6     * task.attempt, 'cpus'    ) }
+        memory = { check_max( 36.GB * task.attempt, 'memory'  ) }
+        time   = { check_max( 8.h   * task.attempt, 'time'    ) }
     }
     withLabel:process_high {
-        cpus   = { check_max( 12    * task.attempt, 'cpus'   ) }
-        memory = { check_max( 72.GB * task.attempt, 'memory' ) }
-        time   = { check_max( 16.h  * task.attempt, 'time'   ) }
+        cpus   = { check_max( 12    * task.attempt, 'cpus'    ) }
+        memory = { check_max( 72.GB * task.attempt, 'memory'  ) }
+        time   = { check_max( 16.h  * task.attempt, 'time'    ) }
     }
     withLabel:process_long {
-        time   = { check_max( 20.h  * task.attempt, 'time'   ) }
+        time   = { check_max( 20.h  * task.attempt, 'time'    ) }
+    }
+    withLabel:process_high_memory {
+        memory = { check_max( 200.GB * task.attempt, 'memory' ) }
```

Comment:
The above-added spaces are to align everything with this one? If yes I think they need a further space

Reply:
🦅 👀 Fixed as part of https://github.com/nf-core/viralrecon/pull/230/commits/66ff1c2fef58859609f9ca1a96d2583b8b2a4e13